hugo/content/contribute.md

```diff
@@ -3,83 +3,239 @@ title: Contribute
+The best way to get into Mumble development is to link up with us on IRC ([#mumble-dev on freenode](irc://chat.freenode.net/mumble-dev)), also available via Matrix (Join: `#freenode_#mumble-dev:matrix.org` or use [Direct Link via Element.io (Web App)](#freenode_#mumble-dev:matrix.org)).
 
-We are thankful for any contribution on translating. Please check the {{< wiki "Language Translation" "Language Translation wiki page" />}} for more information.
+Alternatively you can create or comment on a Pull request or issue.
 
-# Documentation
+### Documentation regarding Development
 
-Mumble and all around it needs documentation. Great documentation for a great program.
+- [Contributing guidelines](https://github.com/mumble-voip/mumble#contributing)
 
-**Website**: This website has a public hugo source repository at [mumble-www](https://github.com/mumble-voip/mumble-www). You can report [issues](https://github.com/mumble-voip/mumble-www/issues), create [requests](https://github.com/mumble-voip/mumble-www/issues), and create pull requests for implemented change suggestions.
+- [Development Documentation on Github](https://github.com/mumble-voip/mumble/tree/master/docs/dev)
 
-**Wiki**: Our [documentation wiki](https://wiki.mumble.info) covers some more advanced topics than this website; for example for server administrators and contributors. Any contributions welcome, be it improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+- [Building Instructions](https://github.com/mumble-voip/mumble#building)
 
-**Wikipedia**: Someone that is not a development team member needs to maintain the [article at Wikipedia](https://en.wikipedia.org/wiki/Mumble_%28software%29). At the moment it’s missing references; try to find some. (This requires more work than just googling). Contributing to or creating a mumble article on non-English Wikipedia are also appreciated. See Promotion section below.
+<!-- Should be implemented once it's ready:
+- (Work-in-Progress) [C++ guidelines & tools](https://github.com/mumble-voip/mumble/discussions/4261) 
 
-**Ice methods**: We have a [generated documentation](http://mumble.sourceforge.net/slice/) for exposed Ice methods. Although probably only developers using the doc will be able to understand it, please give us a note on any inconsistencies, errors or needed clarifications if you find them.
+-->
 
-# Promotion
+### Open issues and ongoing development
+
+Take a look at [issues](https://github.com/mumble-voip/mumble/issues) and [pull requests](https://github.com/mumble-voip/mumble/pulls) on GitHub, especially with the following Labels:
 
-Get the word out. This means more than writing about it on your blog ;)
+<!-- add label for important tasks? -->
```

Comment:
@Krzmbrzl Could you maybe also ask that in one your meetings?

Reply:
Will do

Reply:
Okay so the idea will be that we'll assess new bug reports on how critical we think they are. If they are major, we'll label them has high prio and we want to try to fix them before a new release.
This will apply to bugs only though.

Reply:
Ok, should we mention that label in this part then?

Reply:
Yeah we can mention it.
We did not really conclude on which label it is exactly that we want to use for that... :thinking:

Reply:
Maybe just "important" ?

Reply:
We already have the different `Prio` labels so it would probably make sense to reuse them. But given that we have different priority levels, this contradicts the idea of having a binary choice for this :thinking: 

Maybe it is time to delete the Prio labels and create a new `important` label instead. We would have to look at the issues that currently carry a prio label in order to see if any of those should have the new label instead...

Reply:
I will wait then for your decision.

By the way: Should I add the `bug` label as well?
https://github.com/mumble-voip/mumble/labels/bug

Reply:
Yeah adding the `bug` and `feature-request` label might be a good thing here :+1:

Reply:
added this to the todo list, so i resolve the conversation.